python/mxnet/symbol.py

```diff
@@ -133,13 +143,33 @@ def __copy__(self):
         return self.__deepcopy__(None)
 
     def __deepcopy__(self, _):
+        """Returns a deep copy of the input object.
+
+        This functions takes an object as an input and returns a deep copy of that object
```

Comment:
Suggest rewording as:
This function returns a deep copy of that input object including the current state of all its parameters such as weights, biases, etc.